### DISCUSSION TITLE
Thx and Respect to the creator!

### DISCUSSION THREAD

**Discussion by ghost** [Lissy93/dashy#778] [General]
For me github and everything around it is relatively new but I really have to say I am amazed by the quality of work and the commitment of Lissy93. Thanks and my respect for your high quality work you have in any case! Just the documentation and changelogs are miles better than those of many expensive premium softwares.

Cheers

**Comment by lissy93** [comment]
Thank you @tygital - this 100% brightened my day 🥰
